dsp-user/core/src/test/scala/dsp/user/sharedtestdata/SharedTestData.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.user.sharedtestdata
+
+import dsp.valueobjects.User._
+import dsp.user.domain.User
+import dsp.errors.BadRequestException
+import zio.prelude.Validation
+import dsp.valueobjects.Id
+
+object SharedTestData {
+  val passwordStrength = PasswordStrength.make(12)
```

Comment:
That is a system property coming from AppConfig which is typesafe and does't need to be a value object.

Reply:
I wasn't sure about how this parameter should be handled in tests. So, you would just use a plain int to provide this value to the method `PasswordHash.make()`?